admin_guide/manage_nodes.adoc

```diff
@@ -241,31 +241,49 @@ controller] can be evacuated; the replication controllers create new pods on
 other nodes and remove the existing pods from the specified node(s). Bare pods,
 meaning those not backed by a replication controller, are unaffected by default.
 
-To list pods that will be migrated without actually performing the evacuation,
-use the `--dry-run` option:
+To actually evacuate all or selected pods on one or more nodes:
```

Comment:
To actually evacuate one or more nodes:
[Selecting pods needs `--pod-selector` and this option is not available in 3.5/3.6]

Reply:
@pravisankar Updated. Thank you!